### PR TITLE
Fixes #28242 - Honor randomized ordering when rerunning failed job

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: test/unit/job_invocation_composer_test.rb [hunk 1]
```diff
@@ -811,5 +811,13 @@ class JobInvocationComposerTest < ActiveSupport::TestCase
       assert created.targeting.resolved?
       assert_equal job_invocation.template_invocations_host_ids, created.targeting.host_ids
     end
+
+    it 'handles takes randomized_ordering from the original job invocation when rerunning failed' do
```
Comment (xprazak2) [inline test/unit/job_invocation_composer_test.rb]: Maybe omit 'handles'?

**Comment by xprazak2** [comment]
Works as expected, form has pre-selected randomized ordering when re-running failed.

**Comment by ares** [comment]
Tests are green, @xprazak2 anything else to add?

**Comment by xprazak2** [comment]
No additional comments.

**Comment by ares** [comment]
Thanks @adamruzicka and @xprazak2, merging!

**Pull request merged**